cmd/cloud/webhook.go

```diff
@@ -146,10 +147,10 @@ func executeWebhookListCmd(flags webhookListFlag) error {
 	if flags.outputToTable {
 		table := tablewriter.NewWriter(os.Stdout)
 		table.SetAlignment(tablewriter.ALIGN_LEFT)
-		table.SetHeader([]string{"ID", "OWNER", "URL"})
+		table.SetHeader([]string{"ID", "OWNER", "URL", "HEADERS"})
```

Comment:
```suggestion
		table.SetHeader([]string{"ID", "OWNER", "URL", "HTTP HEADERS"})
```

Nit suggestion: "Headers" is a bit vague, especially considering this is in the "header" of a table :P